solidity/deploy/21_genesis_relay.ts

```diff
@@ -0,0 +1,34 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
```

Comment:
`21_genesis_relay.ts` prefix is now conflicting with `21_authorize_maintainer_proxy_in_bridge.ts`. Since `LightRelay` genesis is something we want to execute only for mainnet, I propose to prefix it with `91_`.

Reply:
Done in c8f8bfb3bab35edf9d7e8ac8b2e863b59db9cf2a:
```
        renamed:    21_genesis_relay.ts -> 91_genesis_relay.ts
        renamed:    22_retarget_relay.ts -> 92_retarget_relay.ts
        renamed:    23_authorize_relay_maintainer.ts -> 93_authorize_relay_maintainer.ts
```